spec/support/contexts/odata.rb

```diff
@@ -20,9 +20,18 @@ def expect_output(expected)
     # Don't worry about trailing newlines.
     expect(response.body.rstrip).to eq(expected.rstrip)
   end
+
+  def expect_output_fixture(filename, forms: [])
+    form_names = forms.map(&:name)
+    form_q_names = forms.map(&:questionings).flatten.map(&:name)
+    get(path)
+    expect(response).to have_http_status(:ok)
+    expect(response.body).to eq(prepare_fixture("odata/#{filename}", form: form_names,
+                                                                     q_name: form_q_names))
+  end
 end
 
-shared_context "odata_with_forms" do
+shared_context "odata_with_basic_forms" do
```

Comment:
By convention we generally use spaces instead of underscores in rspec descriptions. I see the old api specs violate this but they're on the way out.